### ISSUE TITLE
IQmol 2.14.0 does not compile

### ISSUE THREAD

**Issue by susilehtola** [nutjunkie/IQmol#35]
Trying to update the Fedora packages, I get the error
```
MoleculeLayer.C: In member function 'void IQmol::Layer::Molecule::groupSelection()':
MoleculeLayer.C:1396:45: error: no matching function for call to 'IQmol::Layer::Group::Group(IQmol::Layer::PrimitiveList&)'
 1396 |    appendPrimitive(new Layer::Group(selected));
      |                                             ^
In file included from EfpFragmentLayer.h:25,
                 from MoleculeLayer.C:46:
GroupLayer.h:38:10: note: candidate: 'IQmol::Layer::Group::Group(const QString&)'
   38 |          Group(QString const& label = QString());
      |          ^~~~~
GroupLayer.h:38:31: note:   no known conversion for argument 1 from 'IQmol::Layer::PrimitiveList' to 'const QString&'
   38 |          Group(QString const& label = QString());
      |                ~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~
```

root log https://kojipkgs.fedoraproject.org//work/tasks/4237/40314237/root.log (list of installed packages)
build log https://kojipkgs.fedoraproject.org//work/tasks/4237/40314237/build.log

**Comment by susilehtola** [comment]
@nutjunkie any progress?

**Comment by susilehtola** [comment]
IQmol will get withdrawn from Fedora if this issue is not solved.

**Comment by nutjunkie** [comment]
Fixed in v2.14.3

**Comment by susilehtola** [comment]
Thanks, compiles!
